.gitignore

```diff
@@ -0,0 +1,2 @@
+*.pyc
+/db.sqlite3
```

Comment:
I should be able to copy the teachersportal's stuff.

On Tue, Nov 10, 2015 at 3:04 PM, George Schneeloch <notifications@github.com
> wrote:

> In .gitignore
> <https://github.com/mitodl/ccxcon/pull/7#discussion_r44457263>:
>
> > @@ -0,0 +1,2 @@
> > +*.pyc
> > +/db.sqlite3
>
> We should use postgres for our database. Even if this is only for testing,
> the two databases have differences that may complicate testing and
> reproducability. Is it possible to have something we can run using Docker
> or Vagrant to allow for easy local development? I can go over what we did
> for LORE and the teacher's portal if it's helpful
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mitodl/ccxcon/pull/7/files#r44457263>.
>